pkg/ffmpeg/stream_transcode.go

```diff
@@ -83,28 +123,49 @@ var (
 )
 
 type TranscodeOptions struct {
-	StreamType StreamFormat
-	VideoFile  *file.VideoFile
-	Resolution string
-	StartTime  float64
+	StreamType    StreamFormat
+	VideoFile     *file.VideoFile
+	Resolution    string
+	StartTime     float64
+	HardwareAccel bool
```

Comment:
Here passing `HardwareAccel` through `TranscodeOptions` isn't necessary because the config manager is available on `StreamManager.config`. You just need to add `GetTranscodeHardwareAcceleration()` to `StreamManagerConfig` to be able to use it.
If you look at #3491 you'll see I added the `StreamManager` as an argument to `makeStreamArgs` so you'll be able to directly use it from there.

Reply:
It is done